app/api/views/briefs.py

```diff
@@ -1100,10 +1106,14 @@ def upload_brief_rfx_attachment_file(brief_id, slug):
     if not briefs.has_permission_to_brief(current_user.id, brief.id):
         return forbidden('Unauthorised to update brief')
 
-    return jsonify({"filename": s3_upload_file_from_request(request, slug,
-                                                            os.path.join(brief.framework.slug, 'attachments',
-                                                                         'brief-' + str(brief_id)))
-                    })
+    try:
+        filename = s3_upload_file_from_request(
+            request, slug, os.path.join(brief.framework.slug, 'attachments', 'brief-' + str(brief_id))
+        )
+    except Exception as e:
```

Comment:
same here, can the exception logger be used here?

Reply:
I discussed this one with Nathan.  We think it should remain as is since we're using `abort` if an exception is caught.  The error could be client or server side.